src/actions/cursorForward.ts

```diff
@@ -21,8 +21,9 @@ const cursorForward = (state: State) => {
 
   // context view
   let cursorNew, isValidChild
-  if (showContexts) {
-    const contexts = getContextsSortedAndRanked(state, headValue(state, cursor))
+  const cursorValue = headValue(state, cursor)
+  if (showContexts && cursorValue) {
```

Comment:
and here I'm uncertain, as I don't like to put this here, but feels like `contexts[0]` below it already implies we shouldn't go there with a falsy cursorValue, but dunno

Reply:
Similar to above, we want to check explicitly for undefined. Falsey check includes empty string, which we don't want to do. In general, `contexts` should default to `[]` if the thought no longer exists.